sklearn/svm/src/libsvm/svm.cpp

```diff
@@ -2260,7 +2260,7 @@ static void svm_group_classes(const PREFIX(problem) *prob, int *nr_class_ret, in
                 i = j-1;
                 this_label = label[j];
                 this_count = count[j];
-                while(i>=0 && label[i] > this_label)
+                while(i>=0 && label[i] < this_label)
```

Comment:
That is odd. I'm pretty sure I changed exactly that a while ago. Maybe it was in the liblinear code?